using-timescaledb/telemetry.md

```diff
@@ -80,14 +80,25 @@ release, many users have reported that performance issues or bugs
 automatically resolve after updating their version of TimescaleDB.
 
 ## Disabling Telemetry
+
 Although we invite our community to help us keep improving our
-product, we do understand when users would like to disable telemetry. Note that
-disabling telemetry also disables the version checking functionality.
+product, we do understand when users would like to disable
+telemetry. Note that disabling telemetry also disables the version
+checking functionality.
+
+Telemetry is sent on a per-database basis, so users can disable
+telemetry for specific databases or for an entire instance.
 
-Telemetry is sent on a per-database basis, so users can disable telemetry for specific databases or for an entire instance.
+To disable telemetry, it is necessary to both remove the telemetry job
+from the job list and it is turn down the telemetry reporting
+level. Turning down the telemetry reporting level ensures that nothing
+is sent out even if the telemetry job is scheduled.  Removing the
+telemetry job from the jobs list will avoid it being scheduled for
+execution and hence prevent warnings and and errors resulting from
+scheduling an empty job. These two settings can be done independently.
 
-To turn off telemetry for an instance, simply include the following line
-in your `postgresql.conf` file:
+To turn down the telemetry level for an instance, include the
```

Comment:
Actually the GUC is what prevents telemetry from being sent, and should be set if you want to disable it. In addition you can remove the job but the job will be recreated in a dump/restore cycle. So after dump/restore you need to delete it again.

Reply:
Yes, that is true, but do you suggest that we should add a notice about this?